test/e2e/windows/gmsa_full.go

```diff
@@ -77,7 +76,6 @@ const (
 	gmsaCustomResourceName = "gmsa-e2e"
 
 	// gmsaWebhookDeployScriptURL is the URL of the deploy script for the GMSA webook
-	// TODO(wk8): we should pin versions.
 	gmsaWebhookDeployScriptURL = "https://raw.githubusercontent.com/kubernetes-sigs/windows-gmsa/master/admission-webhook/deploy/deploy-gmsa-webhook.sh"
```

Comment:
did this get updated to use the gcr.io images we started building?

Reply:
nope :(
https://github.com/kubernetes-sigs/windows-gmsa/blob/0c81292f090d2aa7abd24cd0f5a62de02ba4a52e/admission-webhook/deploy/deploy-gmsa-webhook.sh#L25

Let me make an issue for that in the gmsa repo

Reply:
https://github.com/kubernetes-sigs/windows-gmsa/issues/70

Reply:
yea, that shouldn't block this PR,